cli-plugins/manager/plugin.go

```diff
@@ -33,7 +33,9 @@ type Plugin struct {
 // is set, and is always a `pluginError`, but the `Plugin` is still
 // returned with no error. An error is only returned due to a
 // non-recoverable error.
-func newPlugin(c Candidate, rootcmd *cobra.Command) (Plugin, error) {
+//
+// nolint: gocyclo
```

Comment:
Can be done in a follow-up, but I think this function should be split into pieces as it reached the `go-cyclo` threshold.